_security-analytics/usage/detectors.md

```diff
@@ -35,6 +35,16 @@ To edit a detector, begin by selecting the link to the detector in the Detector
 After you select the **Alert triggers** tab, you also have the option to add additional alerts for the detector by selecting **Add another alert condition** at the bottom of the page.
 {: .tip }
 
+### Threat intelligence feeds
+
+A threat intelligence feed is a real-time, continuous data stream that gathers information related to risks or threats. A critical piece of information in the tactical threat intelligence feed is called an *indicator of compromise* (IoC). 
```

Comment:
```suggestion
A threat intelligence feed is a real-time, continuous data stream that gathers information related to risks or threats. A piece of information in the tactical threat intelligence feed that suggests your cluster may have been compromised, such as a log-in from an unknown user or location or anomalous activity like an increase in read volume, is called an *indicator of compromise* (IoC). These IoCs can be used by investigators to help isolate security incidents. 
```

Reply:
@natebower: What about this?

Reply:
@Naarcha-AWS I made a couple of edits, but otherwise this is much better. Thanks!